public/scripts/controllers/AlbumsShowController.js

```diff
@@ -0,0 +1,55 @@
+angular
+  .module('tunely')
+  .controller('AlbumsShowController', AlbumsShowController);
+
+AlbumsShowController.$inject = ['$http', '$routeParams'];
+
+function AlbumsShowController ($http, $routeParams) {
+  var vm = this;
+  vm.newSong = {};
+
+  $http({
+    method: 'GET',
+    url: '/api/albums/'+$routeParams.id
+  }).then(function successCallback(json) {
+    vm.album = json.data;
+  });
+
+  vm.editSong = function (song) {
```

Comment:
Should we use named functions within the controller too?  (not currently defined by style-guide)